platform/pulp/tasking/tasks.py

```diff
@@ -96,7 +96,7 @@ def _queue_reserved_task(name, inner_task_id, resource_id, inner_args, inner_kwa
     try:
         celery.tasks[name].apply_async(*inner_args, **inner_kwargs)
```

Comment:
If you update async_args [here](https://github.com/BrnoPCmaniak/pulp/blob/b6e579ca3a8ec7d50bd0127738d2403b5620f7ee/platform/pulp/tasking/tasks.py#L207) to be args


```
 async_args = args
 async_kwargs = {
            "kwargs": kwargs,
 }
```

Then at https://github.com/BrnoPCmaniak/pulp/blob/b6e579ca3a8ec7d50bd0127738d2403b5620f7ee/platform/pulp/tasking/tasks.py#L97
you don't have to unpack and can call apply_async with:
`celery.tasks[name].apply_async(args=inner_args, kwargs=inner_kwargs)`